cmds/magelland/magelland.go

```diff
@@ -18,22 +18,23 @@ import (
 	"github.com/chain4travel/caminogo/utils/logging"
 	"github.com/chain4travel/magellan/api"
 	"github.com/chain4travel/magellan/balance"
+	"github.com/chain4travel/magellan/caching"
 	"github.com/chain4travel/magellan/cfg"
 	"github.com/chain4travel/magellan/db"
 	"github.com/chain4travel/magellan/models"
-	oreliusRpc "github.com/chain4travel/magellan/rpc"
 	"github.com/chain4travel/magellan/services/rewards"
 	"github.com/chain4travel/magellan/servicesctrl"
 	"github.com/chain4travel/magellan/stream"
 	"github.com/chain4travel/magellan/stream/consumers"
 	"github.com/chain4travel/magellan/utils"
 	"github.com/go-sql-driver/mysql"
+	"github.com/golang-migrate/migrate/v4"
 	"github.com/gorilla/rpc/v2"
 	"github.com/gorilla/rpc/v2/json2"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/spf13/cobra"
 
-	"github.com/golang-migrate/migrate/v4"
+	oreliusRpc "github.com/chain4travel/magellan/rpc"
```

Comment:
Just a small one: should be better magellanRpc.
Beside the fact the original name was ortelius (with "t"), it seems that it is only there because I didn't found it when replacing...